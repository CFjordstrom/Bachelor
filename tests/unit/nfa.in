{
#letter -> [a-z]; 
#digit -> [0-9];
#word -> #letter (#letter | #digit)*;
#num -> #digit+;
#atom -> #word;
#atom -> #num;
#exp -> #atom #rest;
#rest -> ;
#rest -> #atom #rest;
}
#atom