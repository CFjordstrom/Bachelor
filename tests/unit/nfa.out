-mindfa
{
#1 -> [0-9] #3 | [a-z] #2;
#2 -> [a-z0-9] #2 | ();
#3 -> [0-9] #3 | ();
}
#1